tracer/qemu_runner.py

```diff
@@ -64,13 +63,15 @@ def __init__(
 
         if type(input) not in (bytes, TracerPoV):
             raise RunnerEnvironmentError("Input for tracing should be either a bytestring or a TracerPoV for CGC PoV file.")
+        if project or argv or library_path:
+            l.warning('Only absolute path is allowed')
```

Comment:
Can you please be more specific what problem you are trying to fix here?

Reply:
Since we execute things in a temporary dir, relative path user input will break things badly.
We can transform the path that user gives into absolute path but we cannot do the same to `argv`. Because we don't know which is a path which is an argument. the same thing applies to `angr.project`. About `library_path`, maybe we can make it absolute.

Reply:
That is true, but... you are currently raising this warning in every single case. This is the sort of thing that should go in a docstring instead.

Reply:
So I think we can give a warning to let user know that we only support absolute path (hopefully temporarily?)

Reply:
If there's one thing we have learned about users, it is that they will report any warnings they don't recognize as bugs.

Reply:
But even if users read that, sometimes they won't be aware they are using relative path(eg. argv in angr.Project) and qemu_runner may just quit without any warning. Because stdout and stderr have been taken for recording and the script will think it's just not working.
What about something like a `warn_once` which gives out this warning for only once?

Reply:
warn_once won't do anything since you typically only do one trace per process. Kyle, please work with me here - this is not a gigantic issue. A better solution would be to make stderr and stdout available. stdout is available rn, stderr is discarded, that could be fixed.